packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -313,33 +308,45 @@ export default async function main(
 
   try {
     // $FlowFixMe
-    const project = getProjectName({argv, nowConfig: localConfig, isFile, paths});
+    const project = getProjectName({
+      argv,
+      nowConfig: localConfig,
+      isFile,
+      paths
+    });
     log(`Using project ${chalk.bold(project)}`);
 
     const createArgs = {
-        name: project,
-        env: deploymentEnv,
-        build: { env: deploymentBuildEnv },
-        forceNew: argv['--force'],
-        quiet,
-        wantsPublic: argv['--public'] || localConfig.public,
-        isFile,
-        type: null,
-        nowConfig: localConfig,
-        regions,
-        meta
+      name: project,
+      target: argv['--target'],
```

Comment:
The `target` should not be set here, since `--target` will be removed eventually, we only set it later with `--prod`

Reply:
But it's not currently removed, right? In the current version of the CLI you can still use the `--target`? Or are we planning this PR as a breaking release? 🤔